tabpfn_client/client.py

```diff
@@ -117,19 +126,25 @@ def predict(self, train_set_uid: str, x_test):
 
         return np.array(response.json()["y_pred"])
 
-    def error_raising(self, response, method_name):
+    def error_raising(self, response, method_name, only_version_check=False):
```

Comment:
If this is a private method, you could prefix it with `_` as a practice, i.e. `self._error_raising`.
Also, `error_raising` sounds vague to me. Could you give it a better name?

Reply:
I just used the the function definition as it was before and expanded it for the version check. But yes, you're right, I will try to improve this. :)
Regarding the version check, it is checked always, right? With the new only_version_check argument we can just decide if we also want to check for other errors.